diff --git a/.gitignore b/.gitignore
index 211d83ff9..306af31a7 100644
--- a/.gitignore
+++ b/.gitignore
@@ -44,3 +44,6 @@ python3123\Lib\site-packages\playwright\driver\node.exe_split
 .git\objects\pack\pack-1398aa3b048d13cbc3fd30d0965fa2cc30dffaa8.pack_split
 pack-1398aa3b048d13cbc3fd30d0965fa2cc30dffaa8.pack
 
+dist/
+backend/
+start_servers.bat
diff --git a/python3123/Lib/site-packages/fastapi/__pycache__/staticfiles.cpython-312.pyc b/python3123/Lib/site-packages/fastapi/__pycache__/staticfiles.cpython-312.pyc
index 3019040f7..31a3b255b 100644
Binary files a/python3123/Lib/site-packages/fastapi/__pycache__/staticfiles.cpython-312.pyc and b/python3123/Lib/site-packages/fastapi/__pycache__/staticfiles.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/starlette/__pycache__/staticfiles.cpython-312.pyc b/python3123/Lib/site-packages/starlette/__pycache__/staticfiles.cpython-312.pyc
index f08579a0d..cf6901c18 100644
Binary files a/python3123/Lib/site-packages/starlette/__pycache__/staticfiles.cpython-312.pyc and b/python3123/Lib/site-packages/starlette/__pycache__/staticfiles.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/__pycache__/__init__.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/__pycache__/__init__.cpython-312.pyc
index 470b24db4..ea570ffe5 100644
Binary files a/python3123/Lib/site-packages/uvicorn/__pycache__/__init__.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/__pycache__/__init__.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/__pycache__/__main__.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/__pycache__/__main__.cpython-312.pyc
index 6caa2e5d4..8c137f65c 100644
Binary files a/python3123/Lib/site-packages/uvicorn/__pycache__/__main__.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/__pycache__/__main__.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/__pycache__/_subprocess.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/__pycache__/_subprocess.cpython-312.pyc
index 25ce7cb0a..6c6c3de54 100644
Binary files a/python3123/Lib/site-packages/uvicorn/__pycache__/_subprocess.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/__pycache__/_subprocess.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/__pycache__/_types.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/__pycache__/_types.cpython-312.pyc
index 24d33ae16..f4170aa5e 100644
Binary files a/python3123/Lib/site-packages/uvicorn/__pycache__/_types.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/__pycache__/_types.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/__pycache__/config.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/__pycache__/config.cpython-312.pyc
index 210a92a61..6e936146f 100644
Binary files a/python3123/Lib/site-packages/uvicorn/__pycache__/config.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/__pycache__/config.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/__pycache__/importer.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/__pycache__/importer.cpython-312.pyc
index c7c1d1029..e71446dcb 100644
Binary files a/python3123/Lib/site-packages/uvicorn/__pycache__/importer.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/__pycache__/importer.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/__pycache__/logging.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/__pycache__/logging.cpython-312.pyc
index 88f99e735..671d1c086 100644
Binary files a/python3123/Lib/site-packages/uvicorn/__pycache__/logging.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/__pycache__/logging.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/__pycache__/main.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/__pycache__/main.cpython-312.pyc
index d656b2078..1e197b857 100644
Binary files a/python3123/Lib/site-packages/uvicorn/__pycache__/main.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/__pycache__/main.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/__pycache__/server.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/__pycache__/server.cpython-312.pyc
index 7671086f5..1a13377ce 100644
Binary files a/python3123/Lib/site-packages/uvicorn/__pycache__/server.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/__pycache__/server.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/lifespan/__pycache__/__init__.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/lifespan/__pycache__/__init__.cpython-312.pyc
index 4266887c3..9f2b42069 100644
Binary files a/python3123/Lib/site-packages/uvicorn/lifespan/__pycache__/__init__.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/lifespan/__pycache__/__init__.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/lifespan/__pycache__/on.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/lifespan/__pycache__/on.cpython-312.pyc
index 4b5feefa9..a6385a9dc 100644
Binary files a/python3123/Lib/site-packages/uvicorn/lifespan/__pycache__/on.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/lifespan/__pycache__/on.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/loops/__pycache__/__init__.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/loops/__pycache__/__init__.cpython-312.pyc
index a72bc072e..aea448da3 100644
Binary files a/python3123/Lib/site-packages/uvicorn/loops/__pycache__/__init__.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/loops/__pycache__/__init__.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/loops/__pycache__/asyncio.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/loops/__pycache__/asyncio.cpython-312.pyc
index 7b4812879..61e586f04 100644
Binary files a/python3123/Lib/site-packages/uvicorn/loops/__pycache__/asyncio.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/loops/__pycache__/asyncio.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/loops/__pycache__/auto.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/loops/__pycache__/auto.cpython-312.pyc
index e8f9e5f16..b93cd0f84 100644
Binary files a/python3123/Lib/site-packages/uvicorn/loops/__pycache__/auto.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/loops/__pycache__/auto.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/middleware/__pycache__/__init__.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/middleware/__pycache__/__init__.cpython-312.pyc
index ec5ef2e00..e5edb854b 100644
Binary files a/python3123/Lib/site-packages/uvicorn/middleware/__pycache__/__init__.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/middleware/__pycache__/__init__.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/middleware/__pycache__/asgi2.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/middleware/__pycache__/asgi2.cpython-312.pyc
index 901231c86..9e350fc68 100644
Binary files a/python3123/Lib/site-packages/uvicorn/middleware/__pycache__/asgi2.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/middleware/__pycache__/asgi2.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/middleware/__pycache__/message_logger.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/middleware/__pycache__/message_logger.cpython-312.pyc
index 5a5f901f3..9e297ce49 100644
Binary files a/python3123/Lib/site-packages/uvicorn/middleware/__pycache__/message_logger.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/middleware/__pycache__/message_logger.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/middleware/__pycache__/proxy_headers.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/middleware/__pycache__/proxy_headers.cpython-312.pyc
index 450efeb49..78ee19371 100644
Binary files a/python3123/Lib/site-packages/uvicorn/middleware/__pycache__/proxy_headers.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/middleware/__pycache__/proxy_headers.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/middleware/__pycache__/wsgi.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/middleware/__pycache__/wsgi.cpython-312.pyc
index 620cdf6de..bc8381e89 100644
Binary files a/python3123/Lib/site-packages/uvicorn/middleware/__pycache__/wsgi.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/middleware/__pycache__/wsgi.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/protocols/__pycache__/__init__.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/protocols/__pycache__/__init__.cpython-312.pyc
index e89d4ffd0..38e613f3d 100644
Binary files a/python3123/Lib/site-packages/uvicorn/protocols/__pycache__/__init__.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/protocols/__pycache__/__init__.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/protocols/__pycache__/utils.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/protocols/__pycache__/utils.cpython-312.pyc
index 8564ce442..1ce482fac 100644
Binary files a/python3123/Lib/site-packages/uvicorn/protocols/__pycache__/utils.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/protocols/__pycache__/utils.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/protocols/http/__pycache__/__init__.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/protocols/http/__pycache__/__init__.cpython-312.pyc
index 4516d3117..bd110e3cc 100644
Binary files a/python3123/Lib/site-packages/uvicorn/protocols/http/__pycache__/__init__.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/protocols/http/__pycache__/__init__.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/protocols/http/__pycache__/auto.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/protocols/http/__pycache__/auto.cpython-312.pyc
index 4e9644c82..421779029 100644
Binary files a/python3123/Lib/site-packages/uvicorn/protocols/http/__pycache__/auto.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/protocols/http/__pycache__/auto.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/protocols/http/__pycache__/flow_control.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/protocols/http/__pycache__/flow_control.cpython-312.pyc
index 232b2b7e1..a9359d390 100644
Binary files a/python3123/Lib/site-packages/uvicorn/protocols/http/__pycache__/flow_control.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/protocols/http/__pycache__/flow_control.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/protocols/http/__pycache__/h11_impl.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/protocols/http/__pycache__/h11_impl.cpython-312.pyc
index 8f4752810..c79a99c70 100644
Binary files a/python3123/Lib/site-packages/uvicorn/protocols/http/__pycache__/h11_impl.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/protocols/http/__pycache__/h11_impl.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/protocols/websockets/__pycache__/__init__.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/protocols/websockets/__pycache__/__init__.cpython-312.pyc
index 088479a88..0f9f8ed29 100644
Binary files a/python3123/Lib/site-packages/uvicorn/protocols/websockets/__pycache__/__init__.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/protocols/websockets/__pycache__/__init__.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/protocols/websockets/__pycache__/auto.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/protocols/websockets/__pycache__/auto.cpython-312.pyc
index 4b76e6aa1..523df2ba1 100644
Binary files a/python3123/Lib/site-packages/uvicorn/protocols/websockets/__pycache__/auto.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/protocols/websockets/__pycache__/auto.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/protocols/websockets/__pycache__/wsproto_impl.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/protocols/websockets/__pycache__/wsproto_impl.cpython-312.pyc
index 993762b65..48cd48315 100644
Binary files a/python3123/Lib/site-packages/uvicorn/protocols/websockets/__pycache__/wsproto_impl.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/protocols/websockets/__pycache__/wsproto_impl.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/supervisors/__pycache__/__init__.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/supervisors/__pycache__/__init__.cpython-312.pyc
index 826a8c7df..2c9c3df7b 100644
Binary files a/python3123/Lib/site-packages/uvicorn/supervisors/__pycache__/__init__.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/supervisors/__pycache__/__init__.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/supervisors/__pycache__/basereload.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/supervisors/__pycache__/basereload.cpython-312.pyc
index 5e514e7ef..55fee8b45 100644
Binary files a/python3123/Lib/site-packages/uvicorn/supervisors/__pycache__/basereload.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/supervisors/__pycache__/basereload.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/supervisors/__pycache__/multiprocess.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/supervisors/__pycache__/multiprocess.cpython-312.pyc
index 5b9528545..4b47ac78a 100644
Binary files a/python3123/Lib/site-packages/uvicorn/supervisors/__pycache__/multiprocess.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/supervisors/__pycache__/multiprocess.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/supervisors/__pycache__/statreload.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/supervisors/__pycache__/statreload.cpython-312.pyc
index 29e82c6d5..c89989b4a 100644
Binary files a/python3123/Lib/site-packages/uvicorn/supervisors/__pycache__/statreload.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/supervisors/__pycache__/statreload.cpython-312.pyc differ
diff --git a/python3123/Lib/site-packages/uvicorn/supervisors/__pycache__/watchfilesreload.cpython-312.pyc b/python3123/Lib/site-packages/uvicorn/supervisors/__pycache__/watchfilesreload.cpython-312.pyc
index d07213e30..5cca8a7d6 100644
Binary files a/python3123/Lib/site-packages/uvicorn/supervisors/__pycache__/watchfilesreload.cpython-312.pyc and b/python3123/Lib/site-packages/uvicorn/supervisors/__pycache__/watchfilesreload.cpython-312.pyc differ
